modules/test/module-configuration-localization-test/src/testIntegration/java/com/liferay/module/configuration/localization/test/ModuleConfigurationLocalizationTest.java

```diff
@@ -83,7 +83,9 @@ public void testConfigurationLocalization() {
 			sb.append(bundleError);
 		}
 
-		Assert.assertEquals(sb.toString(), 0, sb.index());
+		if (sb.index() != 0) {
+			Assert.assertEquals("Localization Test Passed", sb.toString());
```

Comment:
@lanceji92674957 what is this assert? How can these ever equal? It will always be false.